scripts/install-cni.sh

```diff
@@ -64,6 +64,13 @@ else
 	cni_spec=$(echo ${cni_spec:-} | sed -e "s#@cniBandwidthPlugin##g")
 fi
 
+if [ "${ENABLE_CILIUM_PLUGIN}" == "true" ]
+then
+        cni_spec=$(echo ${cni_spec:-} | sed -e "s#@cniCiliumPlugin#,{\"type\": \"cilium-cni\"}#g")
```

Comment:
cloud you also update the netd.yaml, we are using it as a reference. 

https://github.com/GoogleCloudPlatform/netd/blob/master/netd.yaml#L71

Reply:
Included another commit for this :)